scripts/quickstart.sh

```diff
@@ -361,9 +373,9 @@ function setup {
 
   # Pull the docker images.
   echo -e "\nPull the docker images..."
-  DOCKER_IMAGE_PARITY="$DOCKER_IMAGE_PARITY:$DOCKER_IMAGE_FUSE_PARITY_VERSION"
-  DOCKER_IMAGE_NETSTAT="$DOCKER_IMAGE_NETSTAT:$DOCKER_IMAGE_NET_STATS_VERSION"
-  DOCKER_IMAGE_APP="$DOCKER_IMAGE_APP:$DOCKER_IMAGE_FUSE_APP_VERSION"
+  DOCKER_IMAGE_PARITY="$DOCKER_IMAGE_PARITY$TEST_NET_STRING:$DOCKER_IMAGE_FUSE_PARITY_VERSION"
```

Comment:
Can this be problematic that both main and testnet will use the same docker versions for the containers `DOCKER_IMAGE_FUSE_PARITY_VERSION` ?

Reply:
They will use a different version file, the version file gets overwritten on line 302 if running on testnet